### DISCUSSION TITLE
Export to html should contain parameters

### DISCUSSION THREAD

**Discussion by TomEngelmann** [resend/react-email#1504] [Ideas]
### Goals

1. Include option to export the {{.PARAMETER}} within the html file as well

### Background

Currently I have to work with a backend that sends emails using the GoLang text/template library to parse html files along with some data. 

It is basically the same procedure as the react email example with e.g. Nodemailer
```
const options = {
  from: 'you@example.com',
  to: 'user@gmail.com',
  subject: 'hello world',
  html: emailHtml,
};

await transporter.sendMail(options);
```

However, if I try to export this piece of code:
```
<Text style={paragraph}>Sehr geehrte/r {name},</Text>
```
it will be exported to the following html, leaving the parameter out:
```
<p style="font-size:16px;line-height:26px;margin:16px 0">Sehr geehrte/r <!-- -->,</p>

```
Therefore it does not contain the parameter which was defined in the javascript code. 

### Feature wish
The export includes an option to export it without a comment but instead with the parameter like:
```

 <p style="font-size:16px;line-height:26px;margin:16px 0">Sehr geehrte/r {{.name}},</p>

```

Unfortunately, without this option it requires a lot of manual for our team.

### Proposal

Instead of replacing the parameters with a comment, the parameter could be written with this syntax {{.PARAMETER}}

**Comment by bernardinorafael** [comment]
I was just looking for something similar, great idea
